app/views/answers/_answer_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row field-margin-top">
+ <% if current_user %>
+                <div class='col-md-offset-3'>
+                  <% if current_user.id == answer.user_id %>
+                      <%= form_tag(question_answer_path(@question.id, answer.id), method: :put) do %>
```

Comment:
A form_for here would set the method automatically as long as it maps to a resource.